packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/src/models/fhir/r4/models.ts

```diff
@@ -34,7 +36,7 @@ export interface Resource {
 
 export function resourceSerializer(item: Resource): any {
   return {
-    ...item,
+    ...item.additionalProperties,
```

Comment:
expected and since this is Record of unknow: https://github.com/azure/autorest.typescript/blob/main/packages/typespec-test/test/healthInsights_radiologyinsights/spec/common/resources.tsp#L10.